### DISCUSSION TITLE
Add Video and Storyboard Playback support for the editor

### DISCUSSION THREAD

**Discussion by NoffyNoffeh** [ppy/osu#31046] [Ideas] [answered]
Please, I can't map in such a gray and lonely world. 

Actual reason: It really feels like missing parity for stable. Especially when mapping or modding anything with a music video, or tv size anime, etc.

It seems like a huge overlooked element that video and storyboard display is supported in gameplay like stable, and also displays on the main menu, but will not play at all in the editor. There is also the fact seeking is already supported during autoplay for gameplay, including the video/storyboard display. Even if the storyboard can not be edited at this time, displaying it would go a long way.

This shouldn't require full design editor support right away, at minimum a location to add video and adjust its offset, and an editor option to toggle visbility for video/storyboard like stable has should be enough for this post.

![image](https://github.com/user-attachments/assets/6dba5c2c-1c2e-400e-8c40-c21d845bfd57)

I could not find any open issue or ideas threads for this, besides it being mentioned in passing in a general editor feedback thread.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/31144

**Reply by ziv-vy** [reply]
Only partially fixed. You still can't drag videos.

**Reply by bdach** [reply]
and is the issue closed? it's not. so there's nothing to see here.

**Reply by ziv-vy** [reply]
Oh, my bad.
